src/test/local/codecept.conf.js

```diff
@@ -13,27 +13,27 @@ exports.config = {
             user: BROWSERSTACK_USERNAME,
             key: BROWSERSTACK_ACCESS_KEY,
             browser: 'chrome',
```

Comment:
browserName?

Reply:
not here. codecept is not accepting browserName. this is codecept syntax

Reply:
Capabilities object is w3c